src/screens/Inventory/components/InventoryInfo.tsx

```diff
@@ -64,17 +69,17 @@ function InventoryInfo(props: InventoryInfoProps) {
           <Typography variant="h2" color="textPrimary">
             {product.name}
           </Typography>
-          <Typography variant="caption">Last Updated</Typography>
+          <Typography variant="caption">{intl(words.last_updated_date)}</Typography>
           <Typography variant="caption">{lastUpdated}</Typography>
```

Comment:
Should this be `intl(words.last_updated_date, lastUpdated)` instead?

Reply:
Good catch!